### ISSUE TITLE
AttributeError: 'str' object has no attribute 'get'

### ISSUE THREAD

**Issue by Nowaker** [italux/sentry-prometheus-exporter#16]
Navigating to http://0.0.0.0:9790/metrics/ after starting the exporter gets me `AttributeError: 'str' object has no attribute 'get'`

```
% docker run --name sentry-exporter --rm \
-e SENTRY_AUTH_TOKEN="$SENTRY_AUTH_TOKEN" \
-e SENTRY_EXPORTER_ORG="$SENTRY_EXPORTER_ORG" \
-p 9790:9790 italux/sentry-prometheus-exporter

2021-05-13 15:03:11,136 - 1 - INFO - exporter - Starting simple wsgi server...
2021-05-13 15:03:11,137 - 1 - INFO - werkzeug -  * Running on http://0.0.0.0:9790/ (Press CTRL+C to quit)
2021-05-13 15:03:12,924 - 1 - INFO - werkzeug - 172.17.0.1 - - [13/May/2021 15:03:12] "GET / HTTP/1.1" 200 -
2021-05-13 15:03:13,133 - 1 - INFO - werkzeug - 172.17.0.1 - - [13/May/2021 15:03:13] "GET /favicon.ico HTTP/1.1" 404 -
2021-05-13 15:03:15,734 - 1 - INFO - exporter - exporter: cleaning registry collectors...
2021-05-13 15:03:16,427 - 1 - INFO - helpers.prometheus - metadata: sentry organization: None
2021-05-13 15:03:16,427 - 1 - INFO - helpers.prometheus - metadata: no projects specified, loading from API
2021-05-13 15:03:16,653 - 1 - ERROR - flask.app - Exception on /metrics/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/app/exporter.py", line 61, in sentry_exporter
    REGISTRY.register(SentryCollector(sentry, ORG_SLUG, get_metric_config(), PROJECTS_SLUG))
  File "/usr/local/lib/python3.7/site-packages/prometheus_client/registry.py", line 26, in register
    names = self._get_names(collector)
  File "/usr/local/lib/python3.7/site-packages/prometheus_client/registry.py", line 66, in _get_names
    for metric in desc_func():
  File "/app/helpers/prometheus.py", line 216, in collect
    __data = self.__build_sentry_data()
  File "/app/helpers/prometheus.py", line 207, in __build_sentry_data
    api_data = self.__build_sentry_data_from_api()
  File "/app/helpers/prometheus.py", line 128, in __build_sentry_data_from_api
    for project in self.__sentry_api.projects():
  File "/app/libs/sentry.py", line 97, in projects
    "id": proj.get("id"),
AttributeError: 'str' object has no attribute 'get'
2021-05-13 15:03:16,657 - 1 - INFO - werkzeug - 172.17.0.1 - - [13/May/2021 15:03:16] "GET /metrics/ HTTP/1.1" 500 -
```

**Comment by Nowaker** [comment]
I think it's due to an invalid token. After regenerating the token, it went through. The system should provide a friendly error message when the token is invalid, though.

**Comment by italux** [comment]
This issue was resolved on [v0.3.0](https://github.com/italux/sentry-prometheus-exporter/releases/tag/v0.3.0) by #22
